setup.py

```diff
@@ -64,13 +64,15 @@
             "bulk_email_optout = lms.djangoapps.bulk_email.policies:CourseEmailOptout"
         ],
         "lms.djangoapp": [
-            "grades = lms.djangoapps.grades.apps:GradesConfig",
             "ace_common = openedx.core.djangoapps.ace_common.apps:AceCommonConfig",
+            "grades = lms.djangoapps.grades.apps:GradesConfig",
+            "plugins = openedx.core.djangoapps.plugins.apps:PluginsConfig",
```

Comment:
Because signals are connected in `ready()`, will any plugins which end up after this in `INSTALLED_APPS` fail to have their signals connected because that already ran?

Reply:
When I was thinking about this, my conclusion was that since Django has multiple phases in its Startup process, by the time `ready()` is called, all the Django apps would have already been installed and included in `INSTALLED_APPS`.  Thus, the order in which the apps are in `INSTALLED_APPS` shouldn't affect the order in which their signals get connected in `ready()`.  This is possible only because Django has this multi-phase process.

Is this what you were asking?

Reply:
I guess this works because the `INSTALLED_APPS` population happens in the settings module, not the plugin app's `ready()` method.  The tricky part is that Django apps normally register their signal receivers in `ready()`; the way the plugin system is set up, plugins that come before the plugin registration app itself will have any such direct signal connections of their own set up before declared receivers are set up, and ones that come after it will set them up afterwards.  That's a bit confusing, but not necessarily a problem.

Reply:
I suppose I don't understand where the confusion lies.  The `ready()` method gets called only *after* all apps are installed.  By then, all senders and receivers would already have been installed.  This is why Django recommends that all signal connections happen within the `ready()` method.